src/v2/ion/uiSchemaLabelTransformer.js

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { _, loc, $ } from 'okta';
+import Bundles from 'util/Bundles';
+import Logger from 'util/Logger';
+
+const SECURITY_QUESTION_PREFIX = 'enroll-authenticator.security_question.credentials.questionKey.';
+
+const I18N_OVERRIDE_MAPPINGS = {
```

Comment:
should we put this mappings in a separate file and import here for more clarity?

Reply:
`I18N_OVERRIDE_MAPPINGS` is meant to be used in this file only hence feels make more sense to leave within this file for now.

Reply:
Do you expect it can grow larger overtime ?

Reply:
- Could be and until then move it another file if more readable
- What likely to happen is another refactor when backend send i18n key and we have to move it another file then.